lighthouse-cli/bin.ts

```diff
@@ -65,6 +65,7 @@ const cliFlags = yargs
     'port'
   ], 'Configuration:')
   .describe({
+    'disable-cache-clearing': 'Disable clearing of cache and storage before the run',
```

Comment:
other name ideas welcome, cache is more specific than what actually happens but is probably more familiar/understandable to users than a generic `storage`